src/main/java/org/wildfly/security/keystore/AtomicLoadKeyStoreSpi.java

```diff
@@ -46,6 +48,8 @@
 
     @Override
     public void engineLoad(InputStream stream, char[] password) throws IOException, NoSuchAlgorithmException, CertificateException {
+
+        ElytronMessages.tls.tracef("AtomicLoadKeyStore loading:  stream = %s,  password = %b", stream, password);
```

Comment:
Best to avoid logging any passwords, some end users will actually audit log files to ensure no passwords are logged.